discovery-provider/alembic/trigger_sql/handle_user_balance_changes.sql

```diff
@@ -0,0 +1,60 @@
+create or replace function handle_user_balance_change() returns trigger as $$
+declare
+  new_val int;
+  new_tier text;
+  new_tier_value int;
+  previous_tier text;
+  previous_tier_value int;
+begin
+  SELECT label, val into new_tier, new_tier_value
```

Comment:
are you sure into works? i just tried to use that and it didn't. maybe just use as like we do below for tier

Reply:
nvm confused it with as

Reply:
resolved offline, this sql work - was just confusing running it without declaring the variables